cmd/juju/application/storage.go

```diff
@@ -0,0 +1,267 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package application
+
+import (
+	"fmt"
+	"io"
+	"sort"
+
+	"github.com/juju/cmd/v3"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+	"github.com/juju/names/v5"
+
+	"github.com/juju/juju/api/client/application"
+	jujucmd "github.com/juju/juju/cmd"
+	"github.com/juju/juju/cmd/juju/config"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/cmd/output"
+	"github.com/juju/juju/storage"
+)
+
+const (
+	appStorageConfigSummary = "Displays or sets storage directives for an application."
+
+	appStorageConfigDoc = `
+A storage directive describes to the charm how to refer to the storage,
+and where to provision it from and takes the form <storage-name>[=<storage-specification>]; for details
```

Comment:
It's not describing to the charm - a storage directive describes to **juju** how the storage required by a charm should be provisioned.

Reply:
refactored w thanks